test/views/PaymentsIntoPensionsCYAViewSpec.scala

```diff
@@ -0,0 +1,250 @@
+    def expectedCaption(taxYear: Int): String
+
+    val yes: String
+    val no: String
+
+    val reliefAtSource: String
+    val reliefAtSourceAmount: String
+    val oneOff: String
+    val oneOffAmount: String
+    val pensionsTaxReliefNotClaimed: String
+    val retirementAnnuity: String
+    val retirementAnnuityAmount: String
+    val workplacePayments: String
+    val workplacePaymentsAmount: String
+
+    val saveAndContinue: String
+    val error: String
+
+    val reliefAtSourceHidden: String
+    val reliefAtSourceAmountHidden: String
+    val oneOffHidden: String
+    val oneOffAmountHidden: String
+    val pensionsTaxReliefNotClaimedHidden: String
+    val retirementAnnuityHidden: String
+    val retirementAnnuityAmountHidden: String
+    val workplacePaymentsHidden: String
+    val workplacePaymentsAmountHidden: String
+  }
+
+  object CommonExpectedEN extends CommonExpectedResults {
+    def expectedCaption(taxYear: Int): String = s"Payments into pensions for 6 April ${taxYear - 1} to 5 April $taxYear"
+
+    val yes = "Yes"
+    val no = "No"
+
+    val reliefAtSource = "Relief at source (RAS) pension payments"
+    val reliefAtSourceAmount = "Total RAS payments plus tax relief"
+    val oneOff = "One-off RAS payments"
+    val oneOffAmount = "Total one-off RAS payments plus tax relief"
+    val pensionsTaxReliefNotClaimed = "Pensions where tax relief is not claimed"
+    val retirementAnnuity = "Retirement annuity contract payments"
+    val retirementAnnuityAmount = "Total retirement annuity contract payments"
+    val workplacePayments = "Workplace pension payments"
+    val workplacePaymentsAmount = "Total workplace pension payments"
+
+    val saveAndContinue = "Save and continue"
+    val error = "Sorry, there is a problem with the service"
+
+    val reliefAtSourceHidden = "Change whether relief at source pensions payments were made"
+    val reliefAtSourceAmountHidden = "Change total relief at source pensions payments, plus tax relief"
+    val oneOffHidden = "Change whether one-off relief at source pensions payments were made"
+    val oneOffAmountHidden = "Change total one-off relief at source pensions payments, plus tax relief"
+    val pensionsTaxReliefNotClaimedHidden = "Change whether payments were made into a pension where tax relief was not claimed"
+    val retirementAnnuityHidden = "Change whether retirement annuity contract payments were made"
+    val retirementAnnuityAmountHidden = "Change total retirement annuity contract payments"
+    val workplacePaymentsHidden = "Change whether workplace pension payments were made"
+    val workplacePaymentsAmountHidden = "Change total workplace pension payments"
+  }
+
+  object CommonExpectedCY extends CommonExpectedResults {
+    def expectedCaption(taxYear: Int): String = s"Payments into pensions for 6 April ${taxYear - 1} to 5 April $taxYear"
+
+    val yes = "Yes"
+    val no = "No"
+
+    val reliefAtSource = "Relief at source (RAS) pension payments"
```

Comment:
Unrelated to this story, but why is none of this in Welsh?

Reply:
messages.cy is currently in english. We haven't done the translations

Reply:
Understood. It will be in one or two future tickets.